feedthefox/devices/templates/jinja2/device.html

```diff
@@ -29,8 +28,8 @@
       <div class="col-md-3 col-md-offset-0 col-sm-5 col-sm-offset-5">
         <a href="#" class="btn btn-default btn-lg device-builds">
           <span class="glyphicon glyphicon-download img-responsive" aria-hidden="true"></span>
-          <br>
-          Build
+          <p>{{ device.model }}</p>
+          <p>{{ device.comment }}</p>
```

Comment:
Probably we need to check first if there is a comment before displaying anything